README.md

```diff
@@ -1 +1,17 @@
 # GitHub Branch Source plugin for Jenkins
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/github-branch-source)](https://plugins.jenkins.io/github-branch-source)
+[![Changelog](https://img.shields.io/github/v/tag/jenkinsci/github-branch-source-plugin?label=changelog)](https://github.com/jenkinsci/github-branch-source/blob/master/CHANGELOG.md)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/github-branch-source?color=blue)](https://plugins.jenkins.io/github-branch-source)
+[![Contributors](https://img.shields.io/github/contributors/jenkinsci/github-branch-source-plugin.svg)](https://github.com/jenkinsci/github-branch-source/contributors)
+
+## Introduction
+The GitHub Branch Source plugin allows you to create a new project based on the repository structure from one or more 
+GitHub users or organizations. Complete documentation is 
+[hosted by CloudBees](https://docs.cloudbees.com/docs/admin-resources/latest/plugins/github-branch-source).
+
+Older versions of this plugin may not be safe to use. Please review the following warnings before using an older version:
+
+-    [CSRF vulnerability and insufficient permission checks allow capturing credentials](https://jenkins.io/security/advisory/2017-07-10/)
+-    [Users with Overall/Read access can enumerate credential IDs](https://jenkins.io/security/advisory/2017-07-10/)
+-    [Server-side request forgery](https://jenkins.io/security/advisory/2018-06-04/#SECURITY-806)
```

Comment:
Not sure these are still necessary, since the newest of the bullets references a URL which is almost 1.5 years old. But it felt wrong to intentionally leave it out, so, I put it in.

Reply:
Yeah delete these, they are automatically generated on plugins.jenkins.io and show up on the right side of the page, see https://plugins.jenkins.io/script-security for an example.